src/Documenter.jl

```diff
@@ -547,13 +694,22 @@ function deploydocs(;
         return
     end
 
+    local cibranch, pull_request, repo_slug, tag, event_type, finaldeploy
+
+    if haskey(ENV, "CI")
```

Comment:
Perhaps the logic should be swapped around here: if `tag` is set, then do a special local build, otherwise try to automagically figure things out from CI?

Reply:
I also wrote a bit about this in https://github.com/JuliaDocs/Documenter.jl/pull/1067#discussion_r303698669, but the idea was that you could version documentation separate from the package.  I did change that around, a bit, though.